jax/api.py

```diff
@@ -130,11 +131,12 @@ def jit(fun: Callable[..., T],
       arguments to treat as static (compile-time constant). Operations that only
       depend on static arguments will be constant-folded in Python (during
       tracing), and so the corresponding argument values can be any Python
-      object. Calling the jitted function with different values for these
-      constants will trigger recompilation. If the jitted function is called
-      with fewer positional arguments than indicated by ``static_argnums`` then
-      an error is raised. Arguments that are not arrays or containers thereof
-      must be marked as static. Defaults to ().
+      object. Static arguments should be hashable and comparable. Calling the
```

Comment:
I think ["comparable" entails more than just equality comparison](https://docs.python.org/2.0/ref/comparisons.html). Instead, [just saying hashable entails both `__hash__` and `__eq__`](https://docs.python.org/3.1/glossary.html#term-hashable). So I suggest we either just say "Static arguments should be hashable." or else just inline the standard Python definition for explicitness:

```suggestion
      object. Static arguments should be hashable, meaning both ``__hash__`` and ``__eq__` are implemented. Calling the
```

Reply:
Done (manually, copying the suggested edits